### ISSUE TITLE
Retrieving metadata

### ISSUE THREAD

**Issue by tjxie** [christophergandrud/imfr#9]
Dear Christopher,

I tried to retrieve the metadata of the series but failed. Do you know how to solve the problem? Thank you.

```R
imf_metadata('IFS', 'FASMBC_XDC', country = "all", start = 2000,
              end = current_year(), return_raw = FALSE)
  |========================================================================| 100%
Request failed [500]. Retrying in 2 seconds...
  |========================================================================| 100%
Request failed [500]. Retrying in 3 seconds...
  |========================================================================| 100%
Error: Unable to find what you're looking for.
```
Best regards,
TJ

**Comment by christophergandrud** [comment]
Thanks for reporting this @tjxie . There appears to be some issue over on the IMF's side with their API. I contacted them via email and will let you know if I hear anything back.

**Comment by christophergandrud** [comment]
Resolved in b8200890ff25733bdc142e7faabd8a25fd6df47b

Update should be up on CRAN in a day or so.
